os/hal/include/hal_usb_msd.h

```diff
@@ -101,7 +101,7 @@ typedef struct PACKED_VAR {
  * @brief   Represents command status wrapper structure.
  * @details See USB Mass Storage Class Specification.
  */
-typedef struct PACKED_VAR {
+typedef struct {
```

Comment:
As a result of redefinition errors. I don't know if this is the correct way to solve this.